packages/core/src/plugin/default-search-plugin/default-search-plugin.ts

```diff
@@ -166,6 +167,13 @@ export class DefaultSearchPlugin implements OnApplicationBootstrap, OnApplicatio
             }
         });
 
+        this.eventBus.ofType(StockMovementEvent).subscribe(event => {
```

Comment:
This is from the other PR that got merged right? Let's clean that up to only include the actual change relating to ES version.